README.md

```diff
@@ -40,29 +28,75 @@ In the source folder of your application, for Nano S:
 
 ```bash
 $ # docker can be replaced with podman or buildah without sudo
-$ sudo docker run --rm -ti -v "$(realpath .):/app" ledger-app-builder:latest
+$ sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u $USER):$(id -g $USER) ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
 root@656be163fe84:/app# make
```

Comment:
I would have always specify `BOLOS_SDK=$NANOZ_SDK`. This is a bit more verbose, but it's more clear that it must be specified on all jobs, including `scan-build` and `load`.